### PR TITLE
admin mutations

### PR DESCRIPTION
**Pull request by brettski**
Add update and create mutations for admin level only
moved admin delete and cancel from 'me' queries to admin path

### CODE REVIEW

File: src/graphql/resolvers/mutations/adminSessionUpdate.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    originalSession.status,
+    updatedSession.status,
+  );
+  let eventTitle = '';
+
+  if (
+    originalSession.status === 'DRAFT' &&
+    updatedSession.status === 'SUBMITTED'
+  ) {
+    eventTitle = 'newSessionCreatedByAdmin';
```
Comment (csell5) [inline src/graphql/resolvers/mutations/adminSessionUpdate.js:38]: here you're changing the eventName that we're throwing the message against. but that isn't a valid event. Look at the /events/user to see the events we publish.

Comment (csell5) [reply src/graphql/resolvers/mutations/adminSessionUpdate.js:38]: also for this, and others. are we planning to email the end user we assigned  the session too?

File: src/graphql/resolvers/mutations/adminSessionCreate.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      session,
+    }),
+    memberStore(firestore).find(user.sub),
+  ]);
+
+  return { sessionResults, userResults };
+}
+
+function sendUserEvent(sessionResults, userResults, userEvents) {
+  userEvents.emit('newSessionCreatedByAdmin', {
```
Comment (csell5) [inline src/graphql/resolvers/mutations/adminSessionCreate.js:22]: invalid event.. see other comment below.

File: src/graphql/resolvers/mutations/adminSessionUpdate.js (hunk 1 above)
Comment (brettski) [reply src/graphql/resolvers/mutations/adminSessionUpdate.js:38]: @csell5 I am thinking we should add the new event name so we know the difference, do you agree?

I didn't think through if we were going to email them. Thinking now is yes, we should email them.

File: src/graphql/resolvers/mutations/adminSessionCreate.js (hunk 2 above)
Comment (brettski) [reply src/graphql/resolvers/mutations/adminSessionCreate.js:22]: @csell5 again thinking we may want a different eventName for the admin actions to separate them from user events

File: src/graphql/resolvers/mutations/adminSessionUpdate.js (hunk 1 above)
Comment (csell5) [reply src/graphql/resolvers/mutations/adminSessionUpdate.js:38]: I'm not sure adding a new event name solves anything unless we wanted to change how/what we're sending for that event.

If we just want to "know" that could be done at the resolver who emitted the event in the first place.

Comment (brettski) [reply src/graphql/resolvers/mutations/adminSessionUpdate.js:38]: @csell5 My thinking is to know it was an admin mutation over a regular mutation. I suppose there is a better way to handle that. What is the event name `newSessionCreated`?

Comment (brettski) [reply src/graphql/resolvers/mutations/adminSessionUpdate.js]: @csell5 replaced with defined event name. tested and email is sent on create

File: src/graphql/resolvers/mutations/adminSessionCreate.js (hunk 2 above)
Comment (brettski) [reply src/graphql/resolvers/mutations/adminSessionCreate.js]: @csell5 replaced with defined event name. tested and email is sent on create/update

**Pull request merged**